html/login.html

```diff
@@ -1 +1,99 @@
-로그인 화면입니다.
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="/images/favicon.ico" />
+    <link rel="stylesheet" href="/css/account.css" />
+    <!-- pretendard cdn -->
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <!-- normalize.css CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- fontAwsome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css"
+      crossorigin="anonymous"
+    />
+    <title>Login</title>
+  </head>
+  <body>
+    <div class="login-page">
+      <div class="form">
+        <img
+          src="/images/panda-market-logo.png"
+          alt="판다마켓 이미지"
+          class="form-image"
+          onclick="goHomepage()"
+        />
```

Comment:
❗️ 수정요청
페이지 이동의 경우 a 태그 사용을 추천드려요! 그렇게 하시는 것이 JS를 작성하실 필요도 없고 구조도 더 명확해집니다~

```suggestion
        <a href='/'>
          <img
            src="/images/panda-market-logo.png"
            alt="판다마켓"
            class="form-image"
          />
        </a>
```